client/extensions/woocommerce/lib/order-values/test/fixtures/order-with-coupons.js

```diff
@@ -5,27 +5,32 @@ export default {
 		{
 			id: 26,
 			name: 'Mug',
-			product_id: 71,
+			price: 11.64,
+			quantity: 2,
+			subtotal: 31.98,
```

Comment:
This subtotal doesn't make sense to me for the line item price and quantity

Reply:
Yes -- I thought I mentioned it in the PR, but I didn't make it clear. The subtotal is the price of the items before coupons applied, for the total quantity of items - These fake mugs are $15.99, so x2 is 31.98. Then there's the coupon applied, which brings the total down to 23.28. Meanwhile, the price is per-item, and includes the coupon discount. It's... definitely not straightforward.